demo/pubsub_forwarder/README.adoc

```diff
@@ -0,0 +1,62 @@
+= PubSub Forwarder
+
```

Comment:
Excellent documentation for this addition!  Thank you very much!

Reply:
Thanks! I tried to follow the other demos as an example.